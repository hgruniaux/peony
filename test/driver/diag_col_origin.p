//# RUN: peony -fdiagnostics-column-origin=0 %s
//# RUN: peony -fdiagnostics-column-origin=1 %s
//# RUN: peony -fdiagnostics-column-origin=6 %s

fn test() {
    5 + true;
    //~^ ERROR: cannot add 'i32' to 'bool'
}
