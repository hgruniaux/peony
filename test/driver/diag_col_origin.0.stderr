<source>:6:6: error: cannot add 'i32' to 'bool'
    6 |     5 + true;
      |     ~ ^ ~~~~
